CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
```

Comment:
Thinking a bit about this, I'm not 100% sure that we should introduce an advanceable status on gov-ui-kit as ui matches 1:1 the active state. We could, for instance, update app-next to check `status === ACTIVE && sppStageUtils.isApprovalReached()`.
Let's sync on this